.github/workflows/integration.yml

```diff
@@ -36,7 +41,7 @@ jobs:
           ./scripts/run-s2n-quic-integration.sh
 
   rustls-integration:
-    if: github.repository == 'aws/aws-lc-rs'
+    if: github.repository_owner == 'aws'
```

Comment:
how did the prior condition allow `costly CI runs when on non-"aws" repos`? did it run on forks of `aws/aws-lc-rs`?

Reply:
Sorry, I was copy-pasting. I took most of the PR description from a [similar PR](https://github.com/aws/aws-lc/pull/1532) that I posted for the aws-lc repo.  This change only opens the CI to execute on other "aws"-owned repos.